VideoWeb/VideoWeb/ClientApp/src/app/shared/toast/room-closing/room-closing-toast.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container>
+<div class="call" style="display:block;" id="room-closing">
```

Comment:
also might be worth grabbing latest html template from vhtoast

Reply:
remove inline styling

Reply:
Done